src/main/java/worldcities/servlet/ServletResponse.java

```diff
@@ -0,0 +1,13 @@
+package worldcities.servlet;
+
+
+/**
+ * Json response from WorldCitiesServlet
+ *
+ * @param cityName name of city
+ * @param lon string of lon
+ * @param lat string of lat
+ */
+public record ServletResponse(String cityName, String lon, String lat) {
```

Comment:
Consistency is important. If you use `lat` then `lon`, then you should use that ordering everywhere.
They don't need to be Strings, they can stay doubles and GSON will work it out.